src/include/DurianObject.h

```diff
@@ -7,17 +7,21 @@
 
 enum struct DurianType {
     Double,
-    Integer
+    Integer,
+    Boolean
 };
 
 struct DurianObject {
     DurianObject();
     DurianObject(double dval);
     DurianObject(int64_t ival);
+    DurianObject(bool bval);
+    bool isFalse() { return type == DurianType::Boolean && !value.bval; };
```

Comment:
Might it be better to call this `isFalsy()`? That seems to be closer to the intent of the function.

Reply:
i second this